client/dashboard/plugins/scheduled-updates/index.tsx

```diff
@@ -1,19 +1,141 @@
+import { hostingUpdateSchedulesQuery, sitesQuery } from '@automattic/api-queries';
+import { useQuery } from '@tanstack/react-query';
 import { useNavigate } from '@tanstack/react-router';
-import { Button } from '@wordpress/components';
+import { Button, FormToggle } from '@wordpress/components';
+import { DataViews, type Field, filterSortAndPaginate, View } from '@wordpress/dataviews';
 import { __ } from '@wordpress/i18n';
+import { format, fromUnixTime } from 'date-fns';
+import { useState, useMemo } from 'react';
 import {
 	pluginsScheduledUpdatesNewRoute,
 	pluginsScheduledUpdatesRoute,
 } from '../../app/router/plugins';
+import { DataViewsCard } from '../../components/dataviews-card';
 import { PageHeader } from '../../components/page-header';
 import PageLayout from '../../components/page-layout';
+import { SiteIconLink } from '../../sites/site-fields';
+import { ScheduledUpdateRow } from './types';
+
+export const fields: Field< ScheduledUpdateRow >[] = [
+	{
+		id: 'site',
+		type: 'text',
+		label: __( 'Site' ),
+		getValue: ( { item } ) => item.site.name,
+	},
+	{
+		id: 'lastUpdate',
+		type: 'text',
+		label: __( 'Last Update' ),
+		render: ( { item } ) =>
+			item.lastUpdate ? format( fromUnixTime( item.lastUpdate ), 'MMM d, yyyy H:mm' ) : '-',
+	},
+	{
+		id: 'nextUpdate',
+		type: 'text',
+		label: __( 'Next Update' ),
+		render: ( { item } ) => format( fromUnixTime( item.nextUpdate ), 'MMM d, yyyy H:mm' ),
+	},
+	{
+		id: 'schedule',
+		type: 'text',
+		label: __( 'Frequency' ),
+		render: ( { item } ) => item.schedule[ 0 ].toUpperCase() + item.schedule.slice( 1 ),
+	},
+	{
+		id: 'active',
+		type: 'text',
+		label: __( 'Active' ),
+		render: ( { item } ) => <FormToggle checked={ item.active } onChange={ () => {} } />,
+	},
+	{
+		id: 'actions',
+		type: 'text',
+		label: __( 'Actions' ),
+	},
+	{
+		id: 'scheduleId',
+		label: __( 'Schedule' ),
+	},
+	{
+		id: 'icon.ico',
+		label: __( 'Site icon' ),
+		render: ( { item } ) => <SiteIconLink site={ item.site } />,
+		enableSorting: false,
+		enableGlobalSearch: false,
+	},
+];
+
+export const defaultView: View = {
+	type: 'table',
+	perPage: 14,
+	page: 1,
+	search: '',
+	filters: [],
+	titleField: 'site',
+	fields: [ 'lastUpdate', 'nextUpdate', 'schedule', 'active' ],
+	sort: { field: 'site', direction: 'asc' },
+	groupByField: 'scheduleId',
+	mediaField: 'icon.ico',
+	showMedia: true,
+};
 
 export default function PluginsScheduledUpdates() {
+	const [ view, setView ] = useState( defaultView );
 	const navigate = useNavigate( { from: pluginsScheduledUpdatesRoute.fullPath } );
+	const { data: scheduledUpdates, isLoading: isLoadingSchedules } = useQuery(
+		hostingUpdateSchedulesQuery()
```

Comment:
Similarly, if we should add `hostingUpdateSchedulesQuery` to the routing config loader.